### ISSUE TITLE
Minor fix to Jeb

### ISSUE THREAD

**Issue by ScottElliotTaxFoundation** [TaxFoundation/tax-plans#4]
Change text for Jeb to the following:

Income Tax: Itemized Deductions
Eliminates the state and local income tax deduction. Caps all remaining deductions, besides the charitable deduction, at 2 percent of adjusted gross income. Eliminates the Pease limitation on itemized deductions.

Other
Eliminates the deductibility of interest. Eliminates the personal exemption phase-out. Allows second earners to file their tax returns separately. Taxes carried interest at ordinary income rates.

In short, we're moving the sentence about the deductability of interest from the itemized deductions section to the other section.
